### PR TITLE
re-apply blueprint

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
while working on https://github.com/emberjs/ember-render-modifiers/pull/25, 
I noticed that the linting didn't like decorators (wanted to use @tracked for adding tests). 
I figured the last blueprint applied to this addon was pre-octane, so re-applying ember-cli-update hopefully "just works"?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,51 +14,61 @@
...
     "loader.js": "^4.7.0",
+    "npm-run-all": "^4.1.5",
     "prettier": "^1.16.1",
-    "qunit-dom": "^0.8.0",
+    "qunit-dom": "^1.5.0",
     "release-it": "^12.2.1",
     "release-it-lerna-changelog": "^1.0.3"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
```
Comment (NullVoxPopuli) [inline package.json:56]: breaking change

File: package.json [hunk 2]
```diff
@@ -14,51 +14,61 @@
...
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.10.0",
+    "ember-cli-babel": "^7.22.1",
+    "ember-cli-htmlbars": "^5.3.1",
```
Comment (NullVoxPopuli) [inline package.json:28]: this isn't needed

**Comment by NullVoxPopuli** [comment]
looks like the modifier manager polyfill broke somehow?

File: .travis.yml [hunk 3]
```diff
@@ -3,10 +3,9 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "6"
+  - "10"
```
Comment (rwjblue) [inline .travis.yml:6]: Can you do this (along with the package.json engines update) in a stand alone PR? Specifically to call out the breaking change of dropping old node support...

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 3 above)
Comment (NullVoxPopuli) [reply .travis.yml:6]: can do!

Comment (NullVoxPopuli) [reply .travis.yml:6]: https://github.com/emberjs/ember-render-modifiers/pull/28

**Pull request closed without merging**